en/developer_guide/install.rst

```diff
@@ -1,149 +1,117 @@
-Download OpenOrchestra
-======================
+Install OpenOrchestra on your computer
+======================================
 
 Open Orchestra is composed of two projects : *open-orchestra* which is the CMS Backoffice and *open-orchestra-front-demo* which
```

Comment:
Must we talk about projects or products ? @nbouquet : how do we communicate about Open Orchestra ?